gnosis/conditional-tokens-explorer
# xDai support

Closes #747

To test this,

- I have moved some DAI to native xDai with [dai-bridge](https://dai-bridge.poa.network/)  (With 0.01 is enough), there is also a [faucet](https://xdai-faucet.top/)
-  Configure metamask to use xDai in Settings->Networks with `ID: 100`, `RPC:  https://xdai.poanetwork.dev`, `explorer:  https://blockscout.com/poa/xdai`
![Captura de pantalla de 2021-01-12 18-04-13](https://user-images.githubusercontent.com/358059/104374433-a6b25c80-5500-11eb-9027-5fc450f9e9cd.png)
- And to get some ERC20 in xDai [omni-bridge](https://omni.xdaichain.com/), for example DAI to get DAI on xDai

There still some work to be done,
- [x] Configure correctly the CPK to be used in xDai
- [x] Update `.env.example` with the new env vars for xDai
- [x] Add contract for wrapping erc1155 in erc20 to xDai
- [x] Test everything (!). Even filtering because some ERC20 addresses in the graph seem to be the ones in mainnet 

This seems to be working right, the only problem is with the preparation of omen conditions, a change in reality oracle address should solve this.
